pwiz_tools/Skyline/Controls/Startup/StartPage.cs

```diff
@@ -728,6 +746,8 @@ public IFormView ShowingFormView
             }
         }
 
+        public bool TutorialActionsTestDataUrisOnly { get; set; }

```

Comment:
Use an accessor to an IEnumerable<Uri> instead of a switch to tell the StartPage to perform a test.

Reply:
The issue is that enumerating the URIs isn't trivial. Those values are encoded as part of an EventAction and I haven't found any way to get at them at runtime other than actually running the action.

A more extensive refactor would be needed to make those URIs trivially enumerable. I can do that if you feel strongly about it.

Reply:
Good point. It was trickier than I could see from my review. I played around with an Action member that could be set by the test, and felt it achieved my goal of keeping more of the test code in the test, feeling a bit more like when we insert test interfaces in other places to override code like querying UniProt.

Nice test. I tried it with the old URL and it nicely captures the issue and running it indefinitely mostly keeps the full test around 1 second.